add pages (#1)
aspose-pdf-cloud/Aspose.PDF-Products: pdf/net/conversion/_index.md

---
title: PDF File Conversion via Cloud .NET SDK
url: /net/conversion/
description: Convert PDF to Microsoft Word, Excel, PowerPoint Slides, HTML, Images and many other popular formats with just few lines of C# code.
lastmod: "2022-03-17"
---

{{< blocks/products/pf/feature-page-wrap >}}
{{< blocks/products/pf/feature-page-header h1="PDF Document Conversion Cloud .NET SDK" h2="Export PDF to Microsoft Office® Word, Excel, PowerPoint Presentations, Images, HTML and fixed-layout formats" >}}

{{% blocks/products/pf/feature-page-summary %}}
Aspose.PDF Cloud .NET SDK offers your cloud-based C# program to convert PDF to XLS, XLSX, PPTX, DOC, DOCX, MobiXML, JPEG, EMF, PNG, TIFF, GIF, SVG, XML, XPS, FDF, XFDF, EPUB and other formats. It also supports Tex, MHT, PCL, PS, XSL-FO, and Markdown (MD) file formats.
{{% /blocks/products/pf/feature-page-summary  %}}

{{% blocks/products/pf/feature-page-section  h2="Convert PDF to DOCX and More Formats" %}}

> First you should create a new Configuration object with your personal Application Secret and Application Key. Next,
> create an object to connect to the Cloud API. After, you can upload your document file. The next step is to convert the documents directly. Download and save the result.

Reviewer: ```suggestion
+ Create a new Configuration object with your personal Application Secret and Application Key. 
+ Create an API Client object to connect to the Cloud API. 
+ Upload your document file. 
+ Call Cloud API to convert the documents directly. 
+ Download and save the result.
```

    var storageImageFileName = "sample.pdf";

    // Get your ClientId and ClientSecret from https://dashboard.aspose.cloud (free registration required).
    var config = new Configuration(AppSecret, AppKey);
    var pdfApi = new PdfApi(config);

    using (var file = File.OpenRead(localImageFileName))
    {
        var uploadResult = pdfApi.UploadFile(storageImageFileName, file);
    }
    var response = pdfApi.GetPdfInStorageToXlsx(storageImageFileName);
    response.CopyTo(System.IO.File.OpenWrite("sample.xlsx"));
    Console.WriteLine($"PDF to Excel result: {response.Length} bytes");
}
```

{{% /blocks/products/pf/feature-page-code %}}
{{% /blocks/products/pf/feature-page-section %}}

{{< blocks/products/pf/feature-page-options pairs="pdf-to-bmp pdf-to-tiff pdf-to-png pdf-to-gif pdf-to-bmp" >}}

{{% blocks/products/pf/feature-page-section  h2="PDF API Other Supported Conversions" %}}
C# PDF API loads other multiple [formats](https://docs.aspose.cloud/pdf/supported-document-formats/) for manipulation and conversion. API loads the relevant supported format and save into the required format.
{{% /blocks/products/pf/feature-page-section %}}

> {{< blocks/products/pf/feature-page-options pairs=" pdf-to-pptx epub-to-pdf html-to-pdf pdf-to-docx md-to-pdf pcl-to-pdf svg-to-pdf tex-to-pdf ps-to-pdf" >}}

Reviewer: ```suggestion
{{< blocks/products/pf/feature-page-options pairs="pdf-to-pptx epub-to-pdf html-to-pdf pdf-to-docx md-to-pdf pcl-to-pdf svg-to-pdf tex-to-pdf ps-to-pdf" >}}
```
